azurerm/internal/services/sentinel/parse/sentinel_alert_rules_test.go

```diff
@@ -50,6 +50,7 @@ func TestSentinelAlertRuleID(t *testing.T) {
 			Name:  "Correct Case",
 			Input: "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/resGroup1/providers/Microsoft.OperationalInsights/workspaces/space1/providers/Microsoft.SecurityInsights/alertRules/rule1",
 			Expect: &SentinelAlertRuleId{
+				Subscription:  "00000000-0000-0000-0000-000000000000",
```

Comment:
(as above) can we remove this for now, since this should be threaded through by the interface?